docs/Performance.md

```diff
@@ -0,0 +1,242 @@
+# Protobuf Perforamcne
+This benchmark result is tested on workstation with processor of Intel® Xeon® Processor E5-2630 and 32GB RAM
+
+## Parsing performance
+<table>
+<tbody><tr>
+<th rowspan="2"> </th>
+<th colspan="6" rowspan="1">cpp</th>
+<th colspan="6" rowspan="1">java</th>
+<th colspan="6" rowspan="1">python</th>
+</tr>
+<tr>
+<th colspan="2">new</th>
```

Comment:
Add a section to describe these "new","newarena","reuse".

Reply:
Done by add some explanation in column title.